### ISSUE TITLE
Fix links in tutorial

### ISSUE THREAD

**Issue by Yash-10** [poliastro/poliastro#1152]
In [this](https://docs.poliastro.space/en/latest/examples/Propagation%20using%20Cowells%20formulation.html#Cowell's-formulation) tutorial, two links are not rendered correctly.

![Screenshot from 2021-04-05 13-46-45](https://user-images.githubusercontent.com/68844397/113553449-6e10b380-9615-11eb-89b9-306ef804fc6c.png)

Converting those `<a href ...` tags to markdown style links would fix the issue.

**Comment by niharsalunke** [comment]
Hi @Yash-10 

![image](https://user-images.githubusercontent.com/65081055/113622665-ecec0780-967a-11eb-8135-cf05fe765f97.png)
![image](https://user-images.githubusercontent.com/65081055/113622675-eeb5cb00-967a-11eb-9d00-6c6fd5074f21.png)

I think the changes have been made already.

**Comment by astrojuanlu** [comment]
Please share links to actual code instead of screenshots:

https://github.com/poliastro/poliastro/blob/eedc5f1c810b748b7b0bbc719c2147371b0abba3/docs/source/examples/Propagation%20using%20Cowells%20formulation.mystnb#L24

It's interesting that nested HTML got badly translated. Perhaps this is a bug in myst-parser? What do you think @DhruvSondhi ?

**Comment by DhruvSondhi** [comment]
Hello @astrojuanlu, Yes this seems to be off ... Though if we check [here](https://myst-parser.readthedocs.io/en/latest/using/syntax.html#commonmark-inline-tokens), it needs to work as intended ... I will test it out a little bit before I can comment on it as a _**Bug**_ 🤔 ... I will be sharing my findings here as well 😄

**Comment by DhruvSondhi** [comment]
Yes, @astrojuanlu, You are absolutely correct ... It seems to be a bug with `MyST-Parser` ... The notebook is working correctly with all the nested HTML tags (can be seen [here](https://mybinder.org/v2/gh/poliastro/poliastro/eedc5f1c810b748b7b0bbc719c2147371b0abba3) ... It seems to be some issue with the parser for the same 🤔

**Comment by Yash-10** [comment]
Thanks @DhruvSondhi for confirming. I think we should create an issue in the upstream repository or wait to investigate more?

**Comment by DhruvSondhi** [comment]
I am trying to find the cause of the issue, so I would suggest that we wait a little bit before making the required conclusions & hence the issue. I am trying to create a case for the same in another notebook and share my findings for the same here ... After that, we can create an Issue in the Upstream Repository

**Comment by Yash-10** [comment]
Yeah, sure!!

**Comment by astrojuanlu** [comment]
Thanks for confirming @DhruvSondhi ! No rush, let us know when you are done with your investigation

**Comment by niharsalunke** [comment]
@DhruvSondhi based on the link you have shared, I have seen the following markdown syntax for external tags being used:

![image](https://user-images.githubusercontent.com/65081055/113711798-05076980-9703-11eb-90aa-f7f9af490ee0.png)

which is then rendered as:
![image](https://user-images.githubusercontent.com/65081055/113711836-0d5fa480-9703-11eb-808a-be0513442cae.png)

This is the same format which I have used in my follow-up PR.

**Comment by DhruvSondhi** [comment]
Yes, it seems to be an issue ... The `MyST` parser is not able to parse the nested `HTML` Tags. Using anything else other than the `HTML` or native `Markdown` is breaking the building procedure ... I have tried a lot of combinations with `{note}`, `{admonition}` , etc but these first of all don't render correctly in the _Jupyter Notebook_, second of all, it doesn't' all the parser to continue the build. I also tried doing it with other nested HTML tags & it doesn't seem to work ... Another thing that can be tested is by doing something like ` ```html ``` ` but this doesn't render correctly in the notebook ... So, according to me, it is a **BUG** ... Though I will try it once again tomorrow with some other combinations. This is only a problem with the `Cowell's` notebook 🤔
